pkg/controller/job/job_controller_test.go

```diff
@@ -5172,6 +5172,36 @@ func TestFinalizersRemovedExpectations(t *testing.T) {
 	}
 }
 
+func TestActivePodsForRemoval(t *testing.T) {
```

Comment:
This is testing a private function, and I'm not sure if the set of parameters used in this test can really happen, given the invocation context. It would be useful to have an integration test, or unit test for an exported function.

Reply:
Are you saying we should export this function?  Or do you want a higher level test?  

I can replicate it in an integration test as part of this PR if you want that.

Reply:
resolving as I removed this test case.